staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -285,11 +285,16 @@ type KubeletConfiguration struct {
 	// Default: 40
 	// +optional
 	NodeLeaseDurationSeconds int32 `json:"nodeLeaseDurationSeconds,omitempty"`
-	// imageMinimumGCAge is the minimum age for an unused image before it is
+	// ImageMinimumGCAge is the minimum age for an unused image before it is
 	// garbage collected.
 	// Default: "2m"
 	// +optional
 	ImageMinimumGCAge metav1.Duration `json:"imageMinimumGCAge,omitempty"`
+	// ImageMaximumGCAge is the maximum age an image can be unused before it is garbage collected.
+	// The default of this field is 0, which disables it.
+	// Default: 0
+	// +optional
+	ImageMaximumGCAge metav1.Duration `json:"imageMaximumGCAge,omitempty"`
```

Comment:
add validation for this field to the kubelet config validation function:
* must not be negative
* must not be < ImageMinimumGCAge if non-zero?
* anything else?

Reply:
I also added a check for the featuregate